docs/dev/table/functions.md

```diff
@@ -2866,6 +2890,18 @@ STRING.repeat(INT)
       </td>
     </tr> 
 
+    <tr>
+      <td>
+        {% highlight scala %}
+STRING1.regexpReplace(STRING2, STRING3)
+{% endhighlight %}
+      </td>
+       <td>
+         <p>Returns a string resulting from replacing all substrings in <i>string1</i> that match the regular expression <i>string2</i> with replacement <i>string3</i>, if <i>string1</i> or <i>string2</i> or <i>string3</i> is NULL, returns NULL.</p> 
```

Comment:
\<p>Returns a string from \<i>STRING1\</i> with all the substrings that match a regular expression \<i>STRING2\</i> being replaced with \<i>STRING3\</i>.\</p>